src/components/AQIGauge/GaugeSVG.tsx

```diff
@@ -0,0 +1,190 @@
+    }) ?? '';
+  const sensitiveArcAngle = convertToAngle(
+    convertToPercent(sensitive, min, max)
+  );
+  const sensitiveArc =
+    arcGenerator({
+      innerRadius: innerRadius,
+      outerRadius: outerRadius,
+      startAngle: moderateArcAngle,
+      endAngle: sensitiveArcAngle,
+    }) ?? '';
+  const unhealthyArcAngle = convertToAngle(
+    convertToPercent(unhealthy, min, max)
+  );
+  const unhealthyArc =
+    arcGenerator({
+      innerRadius: innerRadius,
+      outerRadius: outerRadius,
+      startAngle: sensitiveArcAngle,
+      endAngle: unhealthyArcAngle,
+    }) ?? '';
+  const veryUnhealthyArcAngle = convertToAngle(
+    convertToPercent(veryUnhealthy, min, max)
+  );
+  const veryUnhealthyArc =
+    arcGenerator({
+      innerRadius: innerRadius,
+      outerRadius: outerRadius,
+      startAngle: unhealthyArcAngle,
+      endAngle: veryUnhealthyArcAngle,
+    }) ?? '';
+
+  // Filled arc to overlay the background arcs
+  const filledAngle = convertToAngle(convertToPercent(aqi, min, max));
+  const filledArc =
+    arcGenerator({
+      innerRadius: innerRadius,
+      outerRadius: outerRadius,
+      startAngle: startAngle,
+      endAngle: filledAngle,
+    }) ?? '';
+
+  // Assign appropriate color
+  const assignColor = (aqi: number) => {
+    let arcColor = 'white';
+    if (aqi <= good) {
+      arcColor = '#08E400';
+    } else if (aqi <= moderate) {
+      arcColor = '#FEFF00';
+    } else if (aqi <= sensitive) {
+      arcColor = '#FF7E02';
+    } else if (aqi <= unhealthy) {
+      arcColor = '#FF0202';
+    } else if (aqi <= veryUnhealthy) {
+      arcColor = '#8F3F97';
+    } else {
+      // Hazardous
+      arcColor = '#7E0224';
+    }
+    return arcColor;
+  };
+
+  const needleColor = '#636360';
+
+  return (
+    <div className="svg">
+      <svg height="150" width="300" viewBox={' -1.05 -1.05 2.1 1.1'}>
+        <path
+          d={goodArc}
+          fill={assignColor(good)}
+          stroke="black"
+          strokeWidth={0.002}
+          fillOpacity="0.3"
+        />
+        <path
+          d={moderateArc}
+          fill={assignColor(moderate)}
+          stroke="black"
+          strokeWidth={0.002}
+          fillOpacity="0.3"
+        />
+        <path
+          d={sensitiveArc}
+          fill={assignColor(sensitive)}
+          stroke="black"
+          strokeWidth={0.002}
+          fillOpacity="0.3"
+        />
+        <path
+          d={unhealthyArc}
+          fill={assignColor(unhealthy)}
+          stroke="black"
+          strokeWidth={0.002}
+          fillOpacity="0.3"
+        />
+        <path
+          d={veryUnhealthyArc}
+          fill={assignColor(veryUnhealthy)}
+          stroke="black"
+          strokeWidth={0.002}
+          fillOpacity="0.3"
+        />
+        <path
+          d={filledArc}
+          fill={assignColor(aqi)}
+          stroke="black"
+          strokeWidth={0.004}
+        />
+        <circle cx={0} cy={0} r={0.08} fill={needleColor} />
```

Comment:
Nit: the round part of the dial is cut off in a slightly weird way at the bottom, might not be worth the effort to change but wanted to note it